device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C32/buffers_defaults_t0.j2

```diff
@@ -44,3 +44,11 @@
         }
     },
 {%- endmacro %}
+
+{% if (SYSTEM_DEFAULTS is defined) and ('tunnel_qos_remap' in SYSTEM_DEFAULTS) and (SYSTEM_DEFAULTS['tunnel_qos_remap']['status'] == 'enabled') %}

```

Comment:
We can also check `SYSTEM_DEFAULTS` in `buffers_config.j2` instead of in each SKU's buffer template. By doing so, a lot of redundant code can be avoided.
The idea is to generate an empty `port_names_list_extra_queues` list when remapping is disabled in `SYSTEM_DEFAULTS`
How do you think? @bingwang-ms

Reply:
Good suggestion. I've updated the `buffers_config.j2` to add the check, and remove the check from `buffer_defaults.j2`. Please help review. Thanks